File: 9_palindrome/Solution.py
from re import S


class Solution(object):
    def isPalindrome(self, x):
        j = [i for i in str(x)]
        j.reverse()
        return str(x) == "".join(j)
        
        
    
    
test = Solution()
test.isPalindrome(123)
        
        

File: 9_palindrome/Solution.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  return (
    String(x) ==
    String(x)
      .split("")
      .map((i) => i)
      .reverse()
      .join("")
  );
};

isPalindrome(123);
